tests/pytests/unit/fileclient/test_fileclient.py

```diff
@@ -8,9 +8,12 @@
 
 import pytest
 
-import salt.utils.files
-from salt import fileclient
-from tests.support.mock import AsyncMock, MagicMock, Mock, patch
+try:
+    import salt.utils.files
+    from salt import fileclient
+    from tests.support.mock import AsyncMock, MagicMock, Mock, patch
+except ImportError:
+    ...
```

Comment:
```suggestion
import salt.utils.files
from salt import fileclient
from tests.support.mock import AsyncMock, MagicMock, Mock, patch
```

Reply:
Lint fails unless I wrap those imports in a try/except

Reply:
Which lint?
Locally? or on CI?

Reply:
On CI

Reply:
Got a link to that CI failure?

Reply:
https://github.com/saltstack/salt/actions/runs/8572914030/job/23496687298?pr=66302

Reply:
There's something else going on, otherwise, the lint stages on the nightly builds of 3006.x would also be failing.

Reply:
If I omit all my changes and just add a comment somewhere in "tests/pytests/unit/fileclient/test_fileclient.py" then Lint fails.   It seems like this file hasn't been touched in a long time and now that I am the first to add to it, new linting rules that didn't get applied before are being expected of me.

idc either way.  Should I ignore the linter and go on in spite of the failures or respect the linter?  Your call.

Reply:
Ignore these lint issues, I'll approve the PR. The fix it to lint would be elsewhere.